product_of_all_other_numbers/product_of_all_other_numbers.py

```diff
@@ -2,15 +2,31 @@
 Input: a List of integers
 Returns: a List of integers
 '''
+
+
 def product_of_all_other_numbers(arr):
     # Your code here
-
-    pass
+    result = []
+    for o in range(len(arr)):
+        for i in range(len(arr)):
```

Comment:
Try not to use a letter o for loop iteration. The convention is to use letters like I,j,k, and so on.